### ISSUE TITLE
Possibility of installing under Alpine

### ISSUE THREAD

**Issue by tomaswarynyca** [openfaas/faasd#192]
## Expected Behaviour
Is there a way to install everything under Alpine?
I ask so that I don't have to use ubuntu
Something smaller to get more out of it 
Will it be possible to remove the systemd requirement?

## Are you a GitHub Sponsor (Yes/No?)
- [x] No

**Comment by alexellis** [comment]
Hi @tomaswarynyca you'd need to fill out the issue template if you'd like support.

**Comment by tomaswarynyca** [comment]
Hi @alexellis thanks for replying
I have just updated the description
I see that faasd is closely linked to systemd
My question is whether it is possible to take an alpine and install faasd there

**Comment by Shikachuu** [comment]
Hey @tomaswarynyca

Currently without any modification, even the install process will fail.
Currently faasd will only run on systems that runs systemd.

About Alpine, first you need to make some changes to run containerd. [Rancher has a nice guide for it.](https://rancher.com/docs/k3s/latest/en/advanced/#additional-preparation-for-alpine-linux-setup)

Then you most probably need to somehow address openrc, the problem is, I wasn't able to find any golang sdk that is offical or at least actively maintained.

Tldr: As for now there is no way to run faasd on Alpine and I don't think it is worth the effort.

**Comment by alexellis** [comment]
Not at this time - systemd is a technology choice.

Perhaps you can use Debian or Ubuntu instead of Alpine Linux on your system? Can you explain the constraints and use-case a little clearer?
